Documentation/contributing/making-changes.rst

```diff
@@ -0,0 +1,256 @@
+.. include:: /substitutions.rst
+.. _making-changes:
+
+Making Changes
```

Comment:
Maybe this should be called "Git Primer" or something similar?

Reply:
@v01d What about "Making Changes Using Git"? I just want to make sure people know this is about tracking the changes they make.